app/helpers/company_pages_analytics_helper.rb

```diff
@@ -0,0 +1,8 @@
+module CompanyPagesAnalyticsHelper
+  def count_pages
+    Page.where(
```

Comment:
ALL pages?
where is tenant?

Reply:
Can I add has_many_though association to tenant model like this:

Tenant:
has_many :pages, through: :company

Company:
belongs_to :tenant
has_many :pages

Page:
belongs_to :tenant

And then, call @current_tenant.pages... in that method?